src/components/title/_fields/name/title-name-field.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
-import { Field } from 'redux-form';
+import { Field } from 'react-final-form';
 import { FormattedMessage } from 'react-intl';
 
 import { TextField } from '@folio/stripes/components';
 
 function validate(value) {
   let errors;
 
-  if (value === '') {
+  if (value === undefined) {
```

Comment:
Should we check for a falsey value here to cover `null`, `undefined`, & `''`?

Reply:
Changed.